plugin/bitrise/envstore.go

```diff
@@ -0,0 +1,8 @@
+package bitrise
+
+type EnvironmentItemModel map[string]string
```

Comment:
do you need to create a custom type for this?

Reply:
let me get rid of this, not really required